src/axom/inlet/examples/arrays.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <iostream>
+#include <unordered_map>
+
+#include "axom/inlet.hpp"
+
+int main() {
+  auto lr = std::make_shared<axom::inlet::LuaReader>();
+
+  // Parse example input file
+  lr->parseString("values = { [1] = 'start', [2] = 'stop', [3] = 'pause' }");
+
+  axom::sidre::DataStore ds;
+
+  // Initialize Inlet
+  auto inlet = std::make_shared<axom::inlet::Inlet>(lr, ds.getRoot());
+
+  // Register the verifier, which will verify the array values
+  auto vals = inlet->getGlobalTable()->addStringArray("values");
+  vals->registerVerifier([&]() -> bool {
+    std::unordered_map<int,std::string> map;
+    if (!vals->getStringArray(map)) {
+      std::cout << "Error: Array not found\n";
+    } else {
+      std::cout << "Map Contents:\n";

```

Comment:
Little strange from a user-perspective to do this in the verify function.  Maybe change this to check that the given values are either "start", "stop", or "pause" to show a semi-real life example.  Then after the verify() call, just loop through the values and print them out like you are now.